solvers/prox_newton.py

```diff
@@ -0,0 +1,34 @@
+from benchopt import BaseSolver, safe_import_context
+
+
+with safe_import_context() as safe_import_ctx:
+    from skglm.experimental import SqrtLasso
+
+
+class Solver(BaseSolver):
+    name = "skglm-prox-newton"
+
+    stopping_strategy = "iteration"
+
+    install_cmd = 'conda'
+    requirements = [
+        'pip:git+https://github.com/scikit-learn-contrib/skglm.git@main'
+    ]
+
+    def set_objective(self, X, y, lmbd):
+        self.X, self.y, self.lmbda = X, y, lmbd
+        self.clf = SqrtLasso(alpha=lmbd, max_iter=2, verbose=0, tol=1e-10)
+        self.run(2)
+
+    @staticmethod
+    def get_next(n_iter):
+        return n_iter + 1
+
+    def run(self, n_iter):
+        if hasattr(self.clf, "solver_"):
```

Comment:
Is there a reason for setting ``max_iter`` through ``solver_`` rather than the estimator directly?

Reply:
Check this out:
```
from skglm.experimental import SqrtLasso
from benchopt.datasets.simulated import make_correlated_data
from numpy.linalg import norm 
import numpy as np

X, y, _ = make_correlated_data()
alpha = norm(X.T @y, ord=np.inf) / norm(y)

clf = SqrtLasso(alpha=alpha / 10, verbose=2, max_iter=1, tol=1e-10)
clf.fit(X, y)


clf.max_iter=10
clf.fit(X, y)
```